### ISSUE TITLE
Plain 9 layers CNN for Benchmark 

### ISSUE THREAD

**Issue by JMingKuo** [zalandoresearch/fashion-mnist#41]
I have several experimental results with different activation function and learning rate with 
CNN architecture like this : C(3,32)-C(3,32)-P2-C(3,64)-C(3,64)-P2-FC64-FC64-S10

|Activation|Learning Rate|MNIST|Fashion MNIST|
|---|---|---|---|
|RELU|0.01|0.9874|0.9883|
|RELU|0.001|0.9388|0.9368|
|SELU|0.01|0.9871|0.9819|
|SELU|0.001|0.9490|0.8202|

For now, my best results are 98.74% and 98.83% for MNIST and Fashion-MNIST, respectively.
The train-val curve could be found in my repository : https://github.com/JMingKuo/fashion-mnist

**Comment by hanxiao** [comment]
thanks for contributing the benchmark. One question, could you check if 
```python
eval_results = mnist_classifier.evaluate(input_fn=eval_input_fn)
```
this line evaluates the complete test data, i.e. 10000 examples? As by default the batch_size of `eval_input_fn` is 100.

**Comment by JMingKuo** [comment]
Hi, I just checked the size of input data by using prediction mode with same input function like follows:
```python
predictions = list(mnist_classifier.predict(input_fn=eval_input_fn))
print("Num of Samples :", len(predictions))
```
and get the output:
```python
Num of Samples : 10000
```
As its document said, it should go through the whole input data one time as the `num_epochs` was set to 1.
If the `batch_size` is 100, it will run 100 times automatically before stopping. 
I think the input function is working correctly.

**Comment by hanxiao** [comment]
Thanks a lot for the confirmation. Btw, did you manually download the data set (four .gz files) and put it to './data/fashion'? Note that Tensorflow will automatically download and use default MNIST if you give an empty folder, see #36. It is hard to notice, as Tensorflow won't throw any error. Could you please plot it to make sure the dataset is the Fashion-MNIST?

Sorry for so many questions. Just want to be sure, because your result currently is the best result in the benchmark.

**Comment by JMingKuo** [comment]
You are right, I just found that I am using wrong data.
I will update the results as soon as I finish the correct one.
Sorry for your inconvenient.

**Comment by hanxiao** [comment]
close this issue for now as the score is on the original MNIST.
